### PR TITLE
(DOCSP-46029) Adds a DR page

### PR DESCRIPTION
**Pull request by sarahsimpers**
DOCSP: https://jira.mongodb.org/browse/DOCSP-46029
Staging: https://deploy-preview-62--docs-atlas-architecture.netlify.app/disaster-recovery/

### CODE REVIEW

File: source/disaster-recovery.txt [hunk 1]
```diff
@@ -0,0 +1,499 @@
...
+
+In the highly unlikely event that an entire cloud provider is
+unavailable, follow these steps to bring your deployment back online:
+
+.. procedure::
+   :style: normal
+   
+   .. step:: Determine when the cloud provider outage began
+
+   .. step:: Determine the alternative cloud provider you would like to deploy your new {+cluster+} on
```
Comment (kanchana-mongodb) [inline source/disaster-recovery.txt:317]:
```suggestion
   .. step:: Identify the alternative cloud provider you would like to deploy your new {+cluster+} on
```

File: source/disaster-recovery.txt [hunk 2]
```diff
@@ -0,0 +1,499 @@
...
+.. _arch-center-atlas-outage:
+
+{+service+} Outage
+``````````````````
+
+In the highly unlikely event that the {+service+} Control Plane is
+unavailable, open a high-priority :atlas:`support ticket </support/#request-support>`. 
+
+Your {+cluster+}
+may still be available and accessible even if the {+atlas-ui+} is
```
Comment (kanchana-mongodb) [inline source/disaster-recovery.txt:348]:
```suggestion
might still be available and accessible even if the {+atlas-ui+} is
```

File: source/disaster-recovery.txt [hunk 3]
```diff
@@ -0,0 +1,499 @@
...
+unavailable.
+
+.. _arch-center-resource-capacity:
+
+Resource Capacity Issues
+````````````````````````
+
+Computational resource (such as disk space, RAM, or CPU) capacity
+issues can result from poor planning or unexpected database traffic.
+This behavior may not be a result of a disaster.
```
Comment (kanchana-mongodb) [inline source/disaster-recovery.txt:358]:
```suggestion
This behavior might not be a result of a disaster. 
```

File: source/disaster-recovery.txt [hunk 4]
```diff
@@ -0,0 +1,499 @@
...
+      To learn how to restore your snapshot, see :atlas:`Restore Your Cluster </backup/cloud-backup/restore-overview/>`.
+
+   .. step:: Point all applications using the failing {+cluster+} to the newly-created {+cluster+}
+
+.. _arch-center-data-deletion:
+
+Deletion of Production Data
+```````````````````````````
+
+Production data may be accidentally deleted due to human error or a bug
```
Comment (kanchana-mongodb) [inline source/disaster-recovery.txt:412]:
```suggestion
Production data might be accidentally deleted due to human error or a bug
```

File: source/disaster-recovery.txt [hunk 5]
```diff
@@ -0,0 +1,499 @@
...
+      You can work with the technical support team during this step.
+
+      If you determine that reverting to an earlier driver version will
+      fix the issue, continue to the next step.
+
+   .. step:: Determine when the issue was introduced and the latest working driver version
+
+   .. step:: Evaluate if any other changes are required to move to an earlier driver version
+      
+      This may include application code or query changes.
```
Comment (kanchana-mongodb) [inline source/disaster-recovery.txt:459]:
```suggestion
      This might include application code or query changes.
```

**Review by kanchana-mongodb — APPROVED** [review]
LGTM % copy

File: source/disaster-recovery.txt [hunk 6]
```diff
@@ -0,0 +1,499 @@
...
+
+- :ref:`arch-center-high-availability`
+- :ref:`arch-center-resiliency`
+- :ref:`arch-center-backups`
+
+Proactive Configuration Recommendations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Use the following proactive configuration recommendations to configure your
+{+service+} deployments and backups for quicker recovery from disasters.
```
[the remark this answers is not in the document]
Comment (sarahsimpers) [reply source/disaster-recovery.txt]: Changed to "to expediate," I don't want to sound like we guarantee that recovery will be "fast," but instead MORE quick than if they didn't do these things

**Pull request merged**